seatunnel-dist/release-docs/LICENSE

```diff
@@ -211,115 +211,163 @@ licenses.
-     (The Apache Software License, Version 2.0) IntelliJ IDEA Annotations (org.jetbrains:annotations:13.0 - http://www.jetbrains.org)
-     (The Apache License, Version 2.0) org.apiguardian:apiguardian-api (org.apiguardian:apiguardian-api:1.1.0 - https://github.com/apiguardian-team/apiguardian)
-     (The Apache Software License, Version 2.0) flink-shaded-asm-7 (org.apache.flink:flink-shaded-asm-7:7.1-13.0 - http://flink.apache.org/flink-shaded-asm-7)
-     (Apache License 2.0) Asynchronous Http Client (org.asynchttpclient:async-http-client:2.5.3 - http://github.com/AsyncHttpClient/async-http-client/async-http-client)
-     (Apache License 2.0) Asynchronous Http Client Netty Utils (org.asynchttpclient:async-http-client-netty-utils:2.5.3 - http://github.com/AsyncHttpClient/async-http-client/async-http-client-netty-utils)
-     (Apache License, Version 2.0) Apache Yetus - Audience Annotations (org.apache.yetus:audience-annotations:0.5.0 - https://yetus.apache.org/audience-annotations)
-     (Apache License, Version 2.0) Apache Yetus - Audience Annotations (org.apache.yetus:audience-annotations:0.7.0 - https://yetus.apache.org/audience-annotations)
-     (The Apache License, Version 2.0) Apache Calcite Avatica (org.apache.calcite.avatica:avatica-core:1.20.0 - https://calcite.apache.org/avatica)
+     (Apache License, Version 2.0) akka-actor (com.typesafe.akka:akka-actor_2.11:2.5.21 - http://akka.io/)
+     (Apache License, Version 2.0) akka-protobuf (com.typesafe.akka:akka-protobuf_2.11:2.5.21 - http://akka.io/)
+     (Apache License, Version 2.0) akka-slf4j (com.typesafe.akka:akka-slf4j_2.11:2.5.21 - http://akka.io/)
+     (Apache License, Version 2.0) akka-stream (com.typesafe.akka:akka-stream_2.11:2.5.21 - http://akka.io/)
+     (Apache License, Version 2.0) An open source Java toolkit for Amazon S3 (net.java.dev.jets3t:jets3t:0.9.0 - http://www.jets3t.org)
+     (The Apache Software License, Version 2.0) Apache Avro IPC (org.apache.avro:avro-ipc:1.7.7 - http://avro.apache.org)
+     (The Apache Software License, Version 2.0) Apache Avro Mapred API (org.apache.avro:avro-mapred:1.8.2 - http://avro.apache.org/avro-mapred)
+     (The Apache License, Version 2.0) Apache Calcite Avatica (org.apache.calcite.avatica:avatica-core:1.17.0 - https://calcite.apache.org/avatica)
      (The Apache License, Version 2.0) Apache Calcite Avatica Metrics (org.apache.calcite.avatica:avatica-metrics:1.20.0 - https://calcite.apache.org/avatica)
+     (Apache License, Version 2.0) Apache Commons BeanUtils (commons-beanutils:commons-beanutils:1.9.3 - https://commons.apache.org/proper/commons-beanutils/)
+     (Apache License, Version 2.0) Apache Commons CLI (commons-cli:commons-cli:1.3.1 - http://commons.apache.org/proper/commons-cli/)
+     (Apache License, Version 2.0) Apache Commons Collections (commons-collections:commons-collections:3.2.2 - http://commons.apache.org/collections/)
```

Comment:
Why do we modify so much meaningless content?
Such changes, especially license changes, are a challenge for any reviewer.

Reply:
I believe it would be so much next time

Reply:
@CalvinKirs I will ensure the licenses are sorted to avoid large diff info to reduce the pressure of code reviewer. And also create a another [PR](https://github.com/apache/incubator-seatunnel/pull/1337) to sort them first, so that we can focus on the script.